### ISSUE TITLE
🔥 Deletion of `toNegativeInt` function in `ResultContext`

### ISSUE THREAD

**Issue by LVMVRQUXL** [kotools/types#475]
> [!IMPORTANT]
> This is an **incompatible binary change** that should be published in a **major** release according to our [evolution principles](https://github.com/kotools/types/blob/main/documentation/declarations-lifecycle.md).

## 📝 Description

We would like to remove the `toNegativeInt` function in `ResultContext`.

## 🔗 Dependencies

This issue is blocked by the following ones:
- [x] #474

## ✅ Checklist

- [ ] Remove the declaration, update the public [API binaries] and the [unreleased changelog].
- [ ] Close this issue as completed and update tracking ones if present.

[api binaries]: https://github.com/kotools/types/blob/main/CONTRIBUTING.md#checking-the-api-binaries
[unreleased changelog]: https://github.com/kotools/types/blob/main/CHANGELOG.md#unreleased

**Comment by LVMVRQUXL** [comment]
Another issue for this task will be created when ready.
